.babelrc.js

```diff
@@ -0,0 +1,66 @@
+/** @format */
+const _ = require( 'lodash' );
+const path = require( 'path' );
+
+const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
+const isCalypsoServer = process.env.CALYPSO_SERVER === 'true';
+const isCalypso = isCalypsoClient || isCalypsoServer;
+
+const modules = isCalypsoClient ? false : 'commonjs'; // only calypso should keep es6 modules
+const codeSplit = require( './server/config' ).isEnabled( 'code-splitting' );
+
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				modules,
+				targets: {
+					browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ],
+				},
+				exclude: [ 'transform-classes', 'transform-template-literals' ], // transform-classes is added manually later.
+			},
+		],
+		'@babel/stage-2',
+		'@babel/react',
+	],
+	plugins: _.compact( [
+		// the two class transforms are to emulate exactly how babel 6 handled classes.
+		// it very slightly diverges from spec but also is more concise.
+		// see: http://new.babeljs.io/docs/en/next/v7-migration.html#babel-plugin-proposal-class-properties
+		[ '@babel/plugin-proposal-class-properties', { loose: true } ],
+		[ '@babel/plugin-transform-classes', { loose: false } ],
+		[ '@babel/plugin-transform-template-literals', { loose: true } ],
+		! isCalypso && 'add-module-exports',
+		isCalypso && [
+			path.join(
+				__dirname,
+				'server',
+				'bundler',
+				'babel',
+				'babel-plugin-transform-wpcalypso-async'
+			),
+			{ async: isCalypsoClient && codeSplit },
+		],
+		'@babel/plugin-proposal-export-default-from',
+		'@babel/plugin-proposal-export-namespace-from',
```

Comment:
FWIW, this is already [included in stage-2](https://github.com/babel/babel/blob/master/packages/babel-preset-stage-2/package.json#L12)

Reply:
Thanks for letting me know! I'll remove it then 👍